source/blog/2017-05-03-minishift-intro.html.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Introducing Minishift - Run OpenShift locally
+date: 2017-05-03 05:45 UTC
+author: budhram
+published: true
+comments: true
+tags: minishift, atomic-developer-bundle, openshift-local
+---
+
+We are happy to introduce you to [Minishift](https://github.com/minishift/minishift) which provides a far better user experience as compared to [Atomic Developer Bundle (ADB)](https://github.com/projectatomic/adb-atomic-developer-bundle). We have shifted our development effort from ADB to Minishift, to improve the user experience as well as to address Vagrant related issues being faced by our users. We'll explain this more in a later blog post.
+
+Minishift is a CLI tool that helps you run OpenShift locally by running a single-node cluster inside a VM. You can try out OpenShift or develop with it, day-to-day, on your local host.
+
+It is actually a single binary which uses [libmachine](https://github.com/docker/machine/tree/master/libmachine) to provision a VM with a local, single node, [OpenShift Origin](https://github.com/openshift/origin) cluster. It uses [OpenShift local cluster management](https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md) underneath to bring up single node cluster.
+
+It's a platform agnostic CLI tool that supports multiple hypervisors like KVM for Linux, Xhyve for Mac OS, HyperV for Windows and VirtualBox in all OS platforms. Detailed instructions about it can be found in the [Minishift documentation](https://github.com/Minishift/minishift#prerequisites).
+
+## Installing Minishift
+
+1. Install your preferred VM platform on your workstation.
+
+    **Note**: You need to install additional docker-machine driver for 'kvm' or 'xhyve'. See [Docker Machine drivers](https://docs.openshift.org/latest/minishift/getting-started/docker-machine-drivers.html) guide to install it properly.
+
+1. Download the archive for your OS from the Minishift [releases page](https://github.com/minishift/minishift/releases/tag/v1.0.0) and unpack it.
+1. Copy the contents of the directory to your preferred location.
+1. Add the minishift binary to your _PATH_ environment variable.
```

Comment:
@jlebon Rendering well for me. 
![screenshot from 2017-05-03 19-55-15](https://cloud.githubusercontent.com/assets/1132451/25665205/824988bc-303a-11e7-8d22-e6704f79b02d.png)